From: John Salatas <jsalatas@gmail.com>
Date: Sat, 9 Sep 2017 22:43:19 -0700
Subject: Qt 5.9: Panel

---
 qml/Panel/Panel.qml | 23 +++++++++++++++++++++--
 1 file changed, 21 insertions(+), 2 deletions(-)

diff --git a/qml/Panel/Panel.qml b/qml/Panel/Panel.qml
index 273214a..1a26531 100644
--- a/qml/Panel/Panel.qml
+++ b/qml/Panel/Panel.qml
@@ -80,7 +80,7 @@ Item {
         property bool revealControls: !greeterShown &&
                                       !applicationMenus.shown &&
                                       !indicators.shown &&
-                                      (decorationMouseArea.containsMouse || menuBarLoader.menusRequested)
+                                      ((decorationMouseArea.containsPointer) || menuBarLoader.menusRequested)
 
         property bool showWindowDecorationControls: (revealControls && PanelState.decorationsVisible) ||
                                                     PanelState.decorationsAlwaysVisible
@@ -100,6 +100,16 @@ Item {
                                         applicationMenus.model
     }
 
+    Timer {
+        id: menuBarTimer
+        interval: 15
+        repeat: false
+        onTriggered: {
+            decorationMouseArea.containsPointer = applicationMenus.state == 'initial' && indicators.state == 'initial'
+        }
+    }
+
+
     Item {
         id: panelArea
         objectName: "panelArea"
@@ -158,6 +168,7 @@ Item {
         MouseArea {
             id: decorationMouseArea
             objectName: "windowControlArea"
+            property bool containsPointer: false
             anchors {
                 left: parent.left
                 right: parent.right
@@ -169,8 +180,16 @@ Item {
                     callHint.showLiveCall();
                 }
             }
-
+            onEntered: {
+                decorationMouseArea.containsPointer = false
+                menuBarTimer.start()
+            }
+            onExited: {
+                decorationMouseArea.containsPointer = false
+                menuBarTimer.stop()
+            }
             onPressed: {
+                menuBarTimer.stop()
                 if (!callHint.visible) {
                     // let it fall through to the window decoration of the maximized window behind, if any
                     mouse.accepted = false;
